rdf_differ/entrypoints/ui/api.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python3
+
+# api.py
+# Date:  18/09/2020
+# Author: Mihai Coșleț
+# Email: coslet.mihai@gmail.com
+
+"""
+Service to consume API
```

Comment:
I think the file name is misleading, a small documentation will help as well, explain the purpose of this file.